packages/nextjs/pages/[contractAddress]/[network].tsx

```diff
@@ -52,16 +51,8 @@ const ContractDetailPage = () => {
       try {
         if (storedAbi && storedAbi.length > 0) {
           setContractData({ abi: storedAbi, address: contractAddress });
-        } else {
-          const abi = await fetchContractABIFromAnyABI(contractAddress, parseInt(network));
-          if (abi) {
-            setContractData({ abi, address: contractAddress });
-          } else {
-            const abiString = await fetchContractABIFromEtherscan(contractAddress, parseInt(network));
-            const abi = JSON.parse(abiString);
-            setContractData({ abi, address: contractAddress });
-          }
```

Comment:
We still might need this logic, because in abi.ninja we allow people to directly land on `[contractAddress]/[network]` page (without needing to come from home page)

For example : 

http://localhost:3000/0xde30da39c46104798bb5aa3fe8b9e0e1f348163f/1

Above is gitcoin contract link if you directly land on the above link ideally it should (since contract is verified) but if we remove this logic it won't work (since we don't have stored abi in Zustand)


am I missing something 😅 ?

Reply:
Oh, how did i not see this! 

Thanks for correcting me!